### PR TITLE
Add Zeus Modules for Adding Spare Parts

### PR DESCRIPTION
**Pull request by jonpas**

### CODE REVIEW

File: addons/zeus/functions/fnc_moduleAddSpareTrack.sqf [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    if (_mouseOverType != "OBJECT") then {
+        [LSTRING(NothingSelected)] call EFUNC(common,displayTextStructured);
+    } else {
+        if !(alive _mouseOverUnit) then {
+            [LSTRING(OnlyAlive)] call EFUNC(common,displayTextStructured);
+        } else {
+            if (getNumber (configFile >> "CfgVehicles" >> "ACE_Track" >> QEGVAR(cargo,size)) > [_mouseOverUnit] call EFUNC(cargo,getCargoSpaceLeft)) then {
+                [LSTRING(OnlyEnoughCargoSpace)] call EFUNC(common,displayTextStructured);
+            } else {
+                ["AddCargoByClass", ["ACE_Track", _mouseOverUnit, 1, true]] call EFUNC(common,localEvent);
```
Comment (jonpas) [inline addons/zeus/functions/fnc_moduleAddSpareTrack.sqf]: Last parameter is for #2614 but even if it doesn't get merged it won't cause issues as it's the last parameter.

File: addons/zeus/config.cpp [hunk 2]
```diff
@@ -25,11 +25,18 @@ class CfgPatches {
...
+            QGVAR(moduleAddSpareTrack),
+            QGVAR(moduleAddSpareWheel)
+        };
+    };
 };
 
 class ACE_Curator {
     GVAR(captives) = "ace_captives";
     GVAR(medical) = "ace_medical";
+    GVAR(cargoAndRepair[]) = {"ace_cargo", "ace_repair"};
```
Comment (SilentSpike) [inline addons/zeus/config.cpp]: uuuuh...what?

Comment (jonpas) [inline addons/zeus/config.cpp]: You wrote `handleZeusUnitAssigned`. :grin:

Comment (SilentSpike) [inline addons/zeus/config.cpp]: I was temporarily confused because it required two modules, but then I got it

Comment (jonpas) [inline addons/zeus/config.cpp]: Yeah, it's a bit weird at first.

**Comment by SilentSpike** [comment]
:+1: Should be fine even with the changes in #2628

**Pull request merged**